content/java/concepts/array-list/terms/retainAll/retainAll.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'retainAll()'
+Description: 'Retains only the elements that are contained in another collection' 
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - Computer Science
+  - Code Foundations
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Collections'
+  - 'Java'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-java'
+  - 'paths/introduction-to-android-with-java'
+---
+
+**retainAll()** method of [**ArrayList**]([url](https://www.codecademy.com/resources/docs/java/array-list)) is used to remove all the ArrayList's elements that are not contained in the specified collection or retains all matching elements in the current ArrayList instance that match all elements from the Collection list passed as a parameter to the method.
+## Syntax
+```
+B.retainAll(Collection c);
+```
+B is an object of the **ArrayList** class. It represents an instance of an ArrayList. 
+Here, B object keeps only the elements that are also present in the specified collection c, 
+effectively removing any elements from B that are not present in c.
```

Comment:
You can add this info in the introductory paragraph itself. Here instead of this sentence, you can describe the parameter in the following format :
```
- `parameter`: Describe the parameter in one line.
```